reference/orders.sf.yml

```diff
@@ -55,12 +55,22 @@ paths:
               type: string
               enum:
                 - lineItems
+                - lineItems.physicalItems
+                - lineItems.physicalItems.socialMedia
+                - lineItems.physicalItems.options
+                - lineItems.physicalItems.categories
```

Comment:
```suggestion
                - lineItems.physicalItems
                - lineItems.physicalItems.giftWrapping
                - lineItems.physicalItems.socialMedia
                - lineItems.physicalItems.options
                - lineItems.physicalItems.categories
```

Reply:
I didn't include gift wrapping as that is always included by default. So adding the parameter wont have an effect